README.md

```diff
@@ -25,39 +25,61 @@ strict [LICENSE](https://github.com/lazybytez/jojo-discord-bot/blob/main/LICENSE
 
 ### Requirements
 
-1. [Go 1.18](https://go.dev/doc/install)
+1. [Go 1.19](https://go.dev/doc/install)
 2. Git
```

Comment:
We require Docker for linting, I'd suggest to list it as an optional dependency.